spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe Friendship, type: :model do
+  describe 'relationships' do
+
+    it { should belong_to :user }
+    it { should belong_to :friend }
+  end
+end
```

Comment:
We can build out this factory + others needed as the need for test objects arise.